src/diffusion_for_multi_scale_molecular_dynamics/models/diffusion_mace.py

```diff
@@ -27,22 +39,41 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return self.linear(x)
 
 
+class LinearClassificationReadoutBlock(torch.nn.Module):
+    """Linear readout for scalar representation."""
+
+    def __init__(self, irreps_in: o3.Irreps, num_classes: int):
+        """Init method."""
+        super().__init__()
+        self.linear = o3.Linear(
+            irreps_in=irreps_in, irreps_out=o3.Irreps(f"{num_classes}x0e")
+        )
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        """Forward."""
+        return self.linear(x)
+
+
 def input_to_diffusion_mace(
-    batch: Dict[AnyStr, torch.Tensor], radial_cutoff: float
+    batch: Dict[AnyStr, torch.Tensor],
+    radial_cutoff: float,
+    num_classes: int = 1,
```

Comment:
I don't think we should have a default value here. It can lead to subtle bugs. The user should be forced to specify how many classes there are.